gen_large.py

```diff
@@ -0,0 +1,26 @@
+start = 1025
```

Comment:
Please don't use non-Rust languages for codegen

Reply:
noted! I will try to rework something with proc-macros